woocommerce/woocommerce-android
# Enhanced magic link request experiment

This PR just adds a 3rd variant to the magic link request experiment. It updates the layout of the login password screen to look as following:

![image](https://user-images.githubusercontent.com/1522856/183405001-2a85e8b9-00d2-4471-9cc8-959142806d97.png)

**To test:**
1. Comment out the contents of the `FirebaseRemoteConfigRepository::fetchRemoteConfig()` and change the `MagicLinkRequestVariant.CONTROL.name` to `MagicLinkRequestVariant.ENHANCED.name` of `defaultValues` in the same class
2. Enter the store address, then email address and tap Continue
3. Verify the new `Log in with magic link` button is displayed as in the screen shot
4. Tap on the button and notice the Magic link request flow is triggered
